B827EBFFFE841277.json

```diff
@@ -0,0 +1,11 @@
+{
+	"gateway_conf": {
+		"gateway_ID": "B827EBFFFE841277",
+		"servers": [ { "server_address": "router.eu.thethings.network", "serv_port_up": 1700, "serv_port_down": 1700, "serv_enabled": true } ],
+		"ref_latitude": 0,
```

Comment:
Could you please enter at least a rough location? Thx!

Reply:
will do